src/status_im/ui/screens/help_center/views.cljs

```diff
@@ -20,10 +20,6 @@
                                         :accessibility-label :faq-button
                                         :action-fn           #(.openURL react/linking "https://wiki.status.im/Questions_around_beta#firstHeading")}]
      [profile.components/settings-item-separator]
-     [profile.components/settings-item {:label-kw            :t/submit-bug
```

Comment:
should we remove this label ?

Reply:
probably, I'd wait for the test team opinion, in case they use it for something

Reply:
let me see